salt/minion.py

```diff
@@ -1415,75 +1233,6 @@ def _refresh_grains_watcher(self, refresh_interval_in_minutes):
                     }
             })
 
-    def _set_tcp_keepalive(self):
-        if hasattr(zmq, 'TCP_KEEPALIVE'):
-            self.socket.setsockopt(
-                zmq.TCP_KEEPALIVE, self.opts['tcp_keepalive']
-            )
-            self.socket.setsockopt(
-                zmq.TCP_KEEPALIVE_IDLE, self.opts['tcp_keepalive_idle']
-            )
-            self.socket.setsockopt(
-                zmq.TCP_KEEPALIVE_CNT, self.opts['tcp_keepalive_cnt']
-            )
-            self.socket.setsockopt(
-                zmq.TCP_KEEPALIVE_INTVL, self.opts['tcp_keepalive_intvl']
-            )
-
-    def _set_monitor_socket(self):
```

Comment:
Already done-- it just moves into the pubchannel (https://github.com/saltstack/salt/pull/22453/files#diff-7e93fb0d8129a1b378ae1a10559936cbR165)